config-model/src/test/java/com/yahoo/searchdefinition/derived/NearestNeighborTestCase.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Please use updated copyright.

Reply:
We don't need to update copyright when we change a file.

Reply:
Yes, but this is a new file.

Reply:
Ah, you're right